hist/hist/src/TFormula.cxx

```diff
@@ -2263,7 +2263,9 @@ void TFormula::ProcessFormula(TString &formula)
          map<TString, Double_t>::iterator constIt = fConsts.find(fun.GetName());
          if (constIt != fConsts.end()) {
             TString pattern = TString::Format("{%s}", fun.GetName());
-            TString value = TString::Format("%lf", (*constIt).second);
+            // #17225: we take into account LC_LOCALE settings for which the decimal separator
+            // is , instead of ., e.g. de_AT.UTF-8
+            TString value = TString::Format("%lf", (*constIt).second).ReplaceAll(",", ".");
```

Comment:
Shouldn't this be dependent on the value of LC_LOCALE?  (in some local we might have `1,200.00` for one thousand two hundreds).

Reply:
actually it seems that there are only 3 decimal separators: comma, period and arabic decimal separator (https://randombits.dev/articles/number-localization/locale-list)

Reply:
Thinking more about it... If we have an arabic decimal separator, it means that we also have arabic digits. At that point, I am unsure whether the decimal separator would be the first problem we hit...

Reply:
Actually, I think we need a larger conversation about local and how to handle them.  It seems odds/confusion to all support local for this single functions.  Until we have this conversation, I recommend we issue an explicit error when encountering a comma here.